packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -421,7 +432,10 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
               fluentTheme={fluentTheme}
             />
           </Stack.Item>
-
+          {
+            /* @conditional-compile-remove(video-background-effects) */
+            <VideoEffectsPane showPanel={showVideoEffectsPane} setShowPanel={setShowVideoEffectsPane} />
```

Comment:
instead of passing the state of the pane from the composite, could we control the render in the CallWithChatComposite with the state here?

Reply:
yeah we can do that but I am not sure that will be the optimal way to open/close the video effects pane. We do need to pass some reference down to the pane as it contains `closebutton` and `lightDismiss`. 
To take example from the existing pane we do pass the callback to the pane:
 
<img width="412" alt="Screenshot 2023-03-21 at 2 45 00 PM" src="https://user-images.githubusercontent.com/97124699/226750554-d15f881c-a548-41ae-b87e-d8d47e0322c7.png">
<img width="453" alt="Screenshot 2023-03-21 at 2 44 45 PM" src="https://user-images.githubusercontent.com/97124699/226750574-d66bb37d-864c-43c4-87d3-6d3b9e3a9eb3.png">